src/NuGet.Core/NuGet.Build.Tasks.Console/MSBuildStaticGraphRestore.cs

```diff
@@ -42,7 +42,8 @@ internal sealed class MSBuildStaticGraphRestore : IDisposable
         {
             "CollectPackageReferences",
             "CollectPackageDownloads",
-            "CollectFrameworkReferences"
+            "CollectFrameworkReferences",
+            "CollectCentralPackageVersions"
```

Comment:
Off topic, but would it be worthwhile adding a comment that changes to this list need to be reflected in `NuGet.Build.Tasks/NuGet.targets` as well?